### PR TITLE
Adds updater for organization limits

### PR DESCRIPTION
**Pull request by SimonDegrafKern**
Gateway https://github.com/code-kern-ai/refinery-gateway/pull/119
Model submodule https://github.com/code-kern-ai/refinery-submodule-model/pull/36
Admin dashboard: https://github.com/code-kern-ai/admin-dashboard/pull/62
Refinery UI: https://github.com/code-kern-ai/refinery-ui/pull/125
Updater https://github.com/code-kern-ai/refinery-updater/pull/28
Config: https://github.com/code-kern-ai/refinery-config/pull/20

### CODE REVIEW

File: upgrade_logic/business_objects/gateway.py [hunk 1]
```diff
@@ -1,8 +1,47 @@
...
+    else:
+        max_rows = get_config_value("max_rows") or 50000
+        max_cols = get_config_value("max_cols") or 25
+        max_char_count = get_config_value("max_char_count") or 100000
+
+        print(
+            f"Add default limit for organizations",
+            flush=True,
+        )
+        general.execute(
```
Comment (JWittmeyer) [inline upgrade_logic/business_objects/gateway.py]: i think these have the wrong indentation.

Comment (SimonDegrafKern) [reply upgrade_logic/business_objects/gateway.py]: done

**Comment by JWittmeyer** [comment]
merge with dev open

- [ ] resolved

**Comment by JWittmeyer** [comment]
I think the config service should also have a pr for the base_config file (services_to_notify)

- [ ] resolved

**Review by JWittmeyer — APPROVED** [review]

**Pull request merged**